qmi/instruments/newport/smc_100cc.py

```diff
@@ -30,3 +44,542 @@ def __init__(self,
+
+        Returns:
+            Encoder increment value.
+        """
+        _logger.info(
+            "Getting encoder increment value of instrument [%s]", self._name)
+        # instrument must be in configuration state to get the encoder increment value.
+        self.reset(controller_address)
+        self.enter_configuration_state(controller_address)
+        res = self._scpi_protocol.ask(
+            self._build_command("SU?"))
+        sleep(self.COMMAND_EXEC_TIME)
+        self._check_error()
+        self.exit_configuration_state(controller_address)
+        return self._actuators[self.controller_address].ENCODER_RESOLUTION / float(res[3:])
+
+    @rpc_method
+    def set_encoder_increment_value(self, value: float, controller_address: Optional[int] = None) -> None:
+        """
+        Set the encoder increment value. By default, to be as close to 1mm as possible.
+        Check the example (SU command) in the doc below to see how the increment value is
+        calculated:
+        https://www.newport.com/mam/celum/celum_assets/np/resources/CONEX-CC_-_Controller_Documentation.pdf?0
+
+        Parameters:
+            value:              Increment value.
+            controller_address: Optional address of the controller that needs to be controlled. By default,
+                                it is set to the initialised value of the controller address.
+        """
+        _logger.info(
+            "Setting encoder increment value of instrument [%s] to [%f]", self._name, value)
+        # instrument must be in configuration state to set the encoder increment value.
+        self.reset(controller_address)
+        self.enter_configuration_state(controller_address)
+        self._scpi_protocol.write(self._build_command(
+            "SU", value))
+        sleep(self.COMMAND_EXEC_TIME)
+        self._check_error()
+        self.exit_configuration_state(controller_address)
+
+    @rpc_method
+    def get_driver_voltage(self, controller_address: Optional[int] = None) -> float:
+        """
+        Get the max. output voltage of the driver to the motor.
+
+        Parameters:
+            controller_address: Optional address of the controller that needs to be controlled. By default,
+                                it is set to the initialised value of the controller address.
+        """
+        _logger.info(
+            "Getting the max. output voltage of the driver to the motor of instrument [%s]", self._name)
+        self.controller_address = controller_address
+        driver_voltage = self._scpi_protocol.ask(
+            self._build_command("DV?"))
+
+        self._check_error()
+        return float(driver_voltage[3:])
+
+    @rpc_method
+    def set_driver_voltage(self, driver_voltage: float, controller_address: Optional[int] = None) -> None:
+        """
+        Set the max. output voltage of the driver to the motor.
+
+        Parameters:
+            driver_voltage:      New profile generator base velocity.
```

Comment:
spacing not consistent in docs

Reply:
Fixed spacing at most of the docstrings.